README.md

```diff
@@ -0,0 +1,77 @@
+| `mattermost_github_integration_secret`         | :heavy_multiplication_x: | `None`                               | Secret that is used to authenticate between GitHub and your server                                               |
+| `mattermost_github_integration_server_hook`    | :heavy_multiplication_x: | `/`                                  | The relative URL where the server listens on                                                                     |
+| `mattermost_github_integration_server_address` | :heavy_multiplication_x: | `0.0.0.0`                            | Address of the server                                                                                            |
+| `mattermost_github_integration_server_port`    | :heavy_multiplication_x: | `5000`                               | Port under which the server listens to webhooks from GitHub                                                      |
+| `mattermost_github_integration_username`       | :heavy_multiplication_x: | `Github`                             | Username to post under in Mattermost                                                                             |
+| `mattermost_github_integration_icon_url`       | :heavy_multiplication_x: | ` `                                  | URL to icon file which should show up in Mattermost                                                              |
+| `mattermost_github_integration_webhook_urls`   | :heavy_multiplication_x: | `[]`                                 | List of webhooks to post to. See below for more information                                                      |
+| `mattermost_github_integration_show_avatars`   | :heavy_multiplication_x: | `true`                               | Show GitHub avatars in the message that is posted                                                                |
+| `mattermost_github_integration_ignore_actions` | :heavy_multiplication_x: | `[]`                                 | List of actions to ignore. See below for more information                                                        |
+| `mattermost_github_integration_user`           | :heavy_multiplication_x: | `www-data`                           | User under which the server should run. The user has to exist                                                    |
+| `mattermost_github_integration_group`          | :heavy_multiplication_x: | `www-data`                           | Group under which the server should run. The group hast to exist                                                 |
+| `mattermost_github_integration_version`        | :heavy_multiplication_x: | `master`                             | What version of the repository to check out. This can be the the literal string HEAD, a branch name, a tag name. |
+
+### Webhooks
+
+| Name      |      Required      | Default | Description                                                                                |
```

Comment:
Why even have a default column in the first place?

Reply:
deleted